packages/wp-now/README.md

```diff
@@ -89,211 +91,234 @@ cd wordpress-plugin-or-theme
--   `wp-now` works with WordPress themes and plugins even if you don't have WordPress installed;
--   `wp-now` allows to easily switch the WordPress version with `wp-now start --wp=version.number`(make sure to replace the `version.number` with the actual WordPress version);
--   `wp-now` does not support Xdebug PHP extension (yet!)
+The `php-wasm` server runs under a VFS where the default `documentRoot` for `wp-now` is always `/var/www/html`. So, a Blueprint executed from a theme named **atlas** would look like this:
 
-Some similarities between Laravel Valet and `wp-now` to be aware of:
+```json
+{
+  "steps": [
+    {
+      "step": "defineWpConfigConsts",
+      "consts": {
+        "WP_DEBUG": true,
+        "WP_DEBUG_LOG": "/var/www/html/wp-content/themes/atlas/example.log"
+      },
+      "method": "define-before-run"
+    }
+  ]
+}
+```
 
--   could be used for non-WordPress projects;
--   deployments are not possible with neither Laravel Valet, nor `wp-now`;
--   possible to switch easily the PHP version;
--   possibility to work on multiple WordPress sites simultaneously
+You can find the `example.log` file in your project's directory.
 
-### wp-env
+## Known issues
+
+-   Running `wp-now start` in 'wp-content' or 'wordpress' mode creates empty directories. See [GitHub issue #32](https://github.com/WordPress/playground-tools/issues/32).
+-   The `wp-now` NPM package may appear to have a random version number. See [GitHub issue #31](https://github.com/WordPress/playground-tools/issues/31).
 
-If you are migrating from `wp-env`, you should be aware of the differences it has with `wp-now`:
+## Comparisons
 
--   `wp-now` supports non-WordPress projects;
--   `wp-now` does not need Docker;
--   `wp-now` does not support Xdebug PHP extension (yet!);
--   `wp-now` does not include Jest for automatic browser testing
+### Laravel Valet
+
+Here's what you need to know if you're migrating from Laravel Valet:
+
+-   `wp-now` handles the entire WordPress installation for you. Just run the `wp-now start` command.
+-   `wp-now` works across all desktop platforms (Mac, Linux, Windows).
+-   `wp-now` does not set up custom domains for you.
+-   `wp-now` works with WordPress themes and plugins even if you don't have WordPress installed (see item #1 above).
+-   `wp-now` allows you to switch the WordPress version with `wp-now start --wp=VERSION_NUMBER`(replace `VERSION_NUMBER` with the actual WordPress version).
+
+### wp-env
 
-Some similarities between `wp-env` and `wp-now` to be aware of:
+Here's what you need to know if you're migrating from `wp-env`:
 
--   no support for custom domains or SSL;
--   `plugin`, `themes` and index modes are available on `wp-env` and `wp-now`;
--   deployments are not possible with neither `wp-env`, nor `wp-now`;
--   possible to switch easily the PHP version
+-   `wp-now` supports non-WordPress projects.
+-   `wp-now` does not require Docker.
+-   `wp-now` does not include Jest for automatic browser testing.
```

Comment:
random discovery: I found a [Jest test](https://github.com/flexseth/playground-docs-updates/blob/trunk/testing/testing-with-jest.md) in the `wordpress-playground` repo

I don't know what to do with these, but they look like proof of possibility..

Reply:
Good catch. There are a bunch of [mentions of Jest across the repo](https://github.com/search?q=repo%3AWordPress%2Fwordpress-playground%20jest&type=code). Maybe this is no longer true, @adamziel?

Reply:
> Hmm, there are a bunch of [references to Jest across the repo](https://github.com/search?q=repo%3AWordPress%2Fwordpress-playground%20jest&type=code). Maybe this is no longer true, @adamziel? Is it even related? 😅

Run `nx test playground-wordpress` to execute the unit tests via [Jest](https://jestjs.io).

https://github.com/WordPress/wordpress-playground/blob/0d37d79e840d09136b3e2474842658b90a90c153/packages/playground/wordpress/README.md?plain=1#L11

It's possible this is for running unit tests on the Playground itself, not for code you're working on?

I feel like if `Jest` is built in, it should be accessible via the command line. But no testing expert...

Adam is out for a couple more days, he'll probably chime in

Reply:
Jest is used in the main Playground repo for a few packages – that's mostly a legacy thing. `wp-now` is [tested using vitest](https://github.com/WordPress/playground-tools/blob/b189b9fa47e830320710e848f186922299c59edf/packages/wp-now/project.json#L61), though.

Reply:
So this is still correct, right?

> `wp-now` does not include Jest for automatic browser testing.

Reply:
I tried to get more context from the PR where this sentence was added, without luck https://github.com/WordPress/wordpress-playground/pull/289/files#diff-e5f37225a045ebfbf5288b320a66601aa3f94e3e07ddb9740fed46b65e4de3cdR152

I'm fine leaving that sentence, because it's true that `wp-now` doesn't use Jest.
But if you are finding it confusing, we can remove it from the docs. I think it refers to running JS tests on your website, similar to [Puppeteer](https://pptr.dev/).

Reply:
Thanks, @sejas! In this case, let's rewrite it to something like:

> `wp-now` does not include e2e/unit testing libraries.

What do you think?

Reply:
Investigating it a bit further, I think we can say:
`wp-now does not include lifecycle scripts.`

Here is the context: https://developer.wordpress.org/block-editor/reference-guides/packages/packages-env/#node-lifecycle-script

@ironnysh , thanks for working on this! 🙌

Reply:
Perfect! Changed it :-)

Reply:
Perfect! Changed it :-)